kustomize/preprod/simple-app/deployment.yaml

```diff
@@ -22,3 +22,15 @@ spec:
               cpu: "250m"
           ports:
             - containerPort: 5000
+          livenessProbe:
+            httpGet:
+              path: /health
```

Comment:
Verify that in your Flask app you expose manually the 2 endpoints are you add a package to expose them